client/static/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Google</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Antic&family=Balsamiq+Sans:wght@700&family=Dancing+Script:wght@500&family=Lato:wght@300;400&family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
+        <script defer src="./app.js"></script>
+    </head>
+    <body onload="pageLoad()">
+        <header>
+            <img src="http://assets.stickpng.com/images/580b57fcd9996e24bc43c51f.png">
+        </header>
+        <section id="formContent">
+        <form>
+            <input type="text" id="searchbar" name="searchbar" placeholder="Enter Search">
+            <button id="searchbutton" type="submit">Search</button>
+        </form>
+        <div>
+            <a onload="pageLoad()" id="lucky">
+                <button  type="text">I'm Feeling Lucky</button>
+            </a>
+        </div>
+        </section>
+
+        <section id="results">
+        </section>
+    </body>
```

Comment:
Nice structure to this html file, makes it easy to understand what goes where :)